### PR TITLE
Add faqs About RPCSX

### PR DESCRIPTION
**Pull request by RyzenDew**
Sadly This is needed in more detail and this is just a start.

if you want anything added let me know and i will.

### CODE REVIEW

File: .github/Faqs.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Faqs About RPCSX
+
+- Does this run triple A games yet?
+  
+  No it does not and won't for a while.
+
+- can RPCSX run Wi-Fi ?
```
Comment (qurious-pixel) [inline .github/Faqs.md:7]: Capitalize "Can"

File: .github/Faqs.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+## Faqs About RPCSX
```
Comment (qurious-pixel) [inline .github/Faqs.md:1]: FAQs (it's an initialism)

File: .github/Faqs.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ 
+- Does RPCSX have a GUI
+  
+  No not yet as it's too early but check #ui for updates on community GUIs
+
+- How do we get firmware for RPCSX ?
+  
+  You dump it decrypted from your own PS4 Console using FTP.
+
+- How can i help develop RPCSX?
```
Comment (qurious-pixel) [inline .github/Faqs.md:19]: Capitalize "I"

File: .github/readme.md [hunk 4]
```diff
@@ -23,6 +23,7 @@ If you want to contribute as a developer, please contact us in the [Discord](htt
 
 * [How to Compile rpcsx](BUILDING.md)
 * [How to Use rpcsx](USAGE.md)
+* [Faqs About RPCSX](Faqs.md)
```
Comment (qurious-pixel) [inline .github/readme.md:26]: [FAQs about RPCSX] and (FAQS.md) to match the style of BUILDING.md and USAGE.md

**Comment by Darkhost1999** [comment]
I have too many review comments pending.
I'll just write here.
You have spaces before punctuation.
Excessive use of abbreviations, or words like can't, don't, and won't.
This is bad for anyone who needs to use a translator and the only fix I can see for this file is to wait for you to be done editing it then to go back through and change all the questions and responses to be legible.

I don't appreciate the answers provided for the Windows questions. The answers provided do not answer any questions rather they avoid the subject entirely dismissing it out of hand.
The order of the questions from top to bottom does not answer any questions someone may obtain from the previous question.
The questions are not sorted.
And this doesn't answer anything such as whether there is any controller support, audio support, or video output support?
Where can I see compatibility for the emulator at this time? What games run or don't run on the emulator at this time?

Even if English is expected I still think someone should be able to copy and paste this file from GitHub into a translator and receive a decent output.

Avoid words like "this", "it", and "thing" because they may not be clear or will get lost in translation as to what you are referring to. Do not let anyone infer what you said. Communicate confidently, strongly, and adequately.

Pro tip. Add the Grammarly extension to your web browser for free and edit this file. Grammarly will fix everything for you and add the commas you're missing.

**Pull request closed without merging**